nilearn/datasets/func.py

```diff
@@ -1138,7 +1138,7 @@ def fetch_abide_pcp(data_dir=None, n_subjects=None, pipeline='cpac',
 
     # Go into specific data folder and url
     data_dir = os.path.join(data_dir, pipeline, strategy)
-    url = '/'.join([url, 'Outputs', pipeline, strategy])
+    url = os.sep.join([url, 'Outputs', pipeline, strategy])
```

Comment:
This is very surprising, I've never seen a URL with '\' to separate elements, even on Windows.